lib/configure.js

```diff
@@ -70,6 +70,9 @@ module.exports = function ToConfigure(app) {
     if (app.config.sockets.subClient) {
       app.config.sockets.adapterOptions.subClient = app.config.sockets.subClient;
     }
+    if (app.config.sockets.subEvent) {
```

Comment:
Looks like we need docs on this one [here](http://sailsjs.org/documentation/reference/configuration/sails-config-sockets)

cc @sgress454